contributing/development/core_and_modules/internal_rendering_architecture.rst

```diff
@@ -177,7 +177,7 @@ Vulkan driver.
 
 **Vulkan context creation:**
 
-- `drivers/vulkan/vulkan_context.cpp <https://github.com/godotengine/godot/blob/4.0/drivers/vulkan/vulkan_context.cpp>`__
+- `drivers/vulkan/vulkan_context.cpp <https://github.com/godotengine/godot/blob/master/drivers/vulkan/vulkan_context.cpp>`__
```

Comment:
These should point to 4.2 otherwise the links will become out of date as time goes on.

Reply:
Done. Note that I've also added links for Direct3D which only work in `master` currently, so I kept those 2 links pointing to `master`.